services/src/main/scala/cromwell/services/cost/GcpCostCatalogService.scala

```diff
@@ -38,28 +39,47 @@ object CostCatalogKey {
   final val expectedSku =
     (".*?N1 Predefined Instance (Core|Ram) .*|" +
       ".*?N2 Custom Instance (Core|Ram) .*|" +
-      ".*?N2D AMD Custom Instance (Core|Ram) .*").r
+      ".*?N2D AMD Custom Instance (Core|Ram) .*|" +
+      "Nvidia Tesla V100 GPU .*|" +
```

Comment:
Currently, since we use the `findFirstIn` function below, omitting the `.*?` does not stop us from matching SKU descriptions that have text before "Nvidia Tesla." Really, as long as this does not affect CPU and RAM SKUs, we should change that function to require the full description to match the regex because those SKUs are for commitment pricing and are not needed.

We should also verify my conclusion that all the GPU SKUs we need look like "Nvidia Tesla T4 GPU running in Paris" or "Nvidia Tesla P4 GPU attached to Spot Preemptible VMs running in Milan."